apps/heureka/src/components/CustomAppShell.jsx

```diff
@@ -0,0 +1,50 @@
+/*
+ * SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from "react"
+import { AppShell, PageHeader, TopNavigation, TopNavigationItem } from "@cloudoperators/juno-ui-components"
+import { useGlobalsActions, useGlobalsActiveNavEntry } from "../hooks/useAppStore"
+import ServicesView from "./services/ServicesView"
+import IssueMatchesView from "./issueMatches/IssueMatchesView"
+import ComponentsView from "./components/ComponentsView"
+import constants from "./shared/constants"
+
+// Configuration for navigation items and their respective components
+const VIEW_CONFIG = {
+  Services: { label: "Services", icon: "dns", component: ServicesView },
+  IssueMatches: { label: "IssueMatches", icon: "autoAwesomeMotion", component: IssueMatchesView },
+  Components: { label: "Components", icon: "autoAwesomeMotion", component: ComponentsView },
+}
+
+const CustomAppShell = ({ children }) => {
+  const { setActiveView, setShowPanel } = useGlobalsActions()
+  const activeView = useGlobalsActiveNavEntry()
+
+  const handleTabSelect = (item) => {
```

Comment:
The event handler still mentions "Tab". I would suggest to name it something more generic that is not a direct reference to the type of navigation you use.
```suggestion
  const handleNavItemChange = (item) => {
```